### ISSUE TITLE
[A11y][Fastpass]Aria-Expanded not allowed on Div Role

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9320]
URL: https://www.nuget.org/packages/manage/upload

Path | #verify-package-block
-- | -- 
Path | #import-readme-block
Path | #submit-block

Fix the following:
ARIA attribute is not allowed: aria-expanded="true"

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
https://accessibilityinsights.io/info-examples/web/aria-allowed-attr

According to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#:~:text=The%20aria-expanded%20indicates%20to%20assistive%20technology%20whether%20the,on%20the%20elements%20that%20own%20expandable%20grouping%20containers. it looks like we should be adding some additional attributes to this div to indicate that it controls certain child elements. https://www.w3.org/WAI/WCAG21/Techniques/failures/F59 this seems like the most relevant failure.

**Comment by camigthompson** [comment]
Verified the issue no longer is flagged by fastpass.
